index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title>personal portfolio</title>
+</head>
+
+<body>
+    <div class="container">
+        <header class="header-flex align-header">
+            <a href="">Tochukwu</a>
+            <div class="menubar">
```

Comment:
- [ ] It would be better if the hamburger button is on the right, and has a nav tag. Currently, though we have margin-right property applied, the effect is minimum and we need to achieve the given design